sdcm/nemesis.py

```diff
@@ -251,19 +251,55 @@ def disrupt_restart_with_resharding(self):
         self.log.info('Set back murmur3_partitioner_ignore_msb_bits value to 12')
         self.target_node.restart_node_with_resharding()
 
-    def _destroy_data(self):
-        # Send the script used to corrupt the DB
-        break_scylla = get_data_dir_path('break_scylla.sh')
-        self.target_node.remoter.send_files(break_scylla,
-                                            "/tmp/break_scylla.sh")
+    def _destroy_data_and_restart_scylla(self):
+
+        ks_cfs = get_non_system_ks_cf_list(loader_node=random.choice(self.loaders.nodes),
+                                           db_node=self.target_node)
+        # try to search files for destroy 10 times
+        search_iterations = 10
+        file_for_destroy = ''
+        while not file_for_destroy and search_iterations > 0:
```

Comment:
instead of the while use `@retry` decorator on the whole method,
raise exceptions when retry needed (Example NoKeyspaceFound(), NoFIilesFoundToDestroy(), etc)

Reply:
fixed